<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor::Internal::TablePropertiesWidget</class>
 <widget class="QWidget" name="Editor::Internal::TablePropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabBorders">
      <attribute name="title">
       <string>Border / Space</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Border</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="borderWidthSpin"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="borderStyleCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Adjust column size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Adjust row size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Margins, Padding and spacing</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0" rowspan="2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="1">
             <widget class="QSpinBox" name="bottomMarginSpin"/>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="topMarginSpin"/>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="leftMarginSpin"/>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="rightMarginSpin"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Margins (pix)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cell spacing</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Cell padding</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="cellSpacingSpin"/>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="cellPaddingSpin"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColors">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="alternateColorsCheck">
         <property name="text">
          <string>Alterning rows color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Border color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Utils::ColorButtonChooser" name="borderColorButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cells background color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Utils::ColorButtonChooser" name="cellBackgroundColorButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ColorButtonChooser</class>
   <extends>QPushButton</extends>
   <header location="global">utils/widgets/colorbuttonchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
